Makefile

```diff
@@ -39,6 +39,10 @@ test-e2e-for-aws-logs:
 test-e2e-for-fluentd:
 	go test -timeout 30m ./e2e -test.v -ginkgo.v --binary "$(AWS_CONTAINERD_LOGGERS_BINARY)" --log-driver "fluentd"
 
+.PHONY: test-e2e-for-splunk
+test-e2e-for-splunk:
+	go test -timeout 30m ./e2e -test.v -ginkgo.v --binary "$(AWS_CONTAINERD_LOGGERS_BINARY)" --log-driver "splunk" --splunk-token ${SPLUNK_TOKEN}
```

Comment:
nit: we should check $SPLUNK_TOKEN and print a message that it should not be empty before this

Reply:
Added the regular gomega check.

Reply:
The error if SPLUNK_TOKEN is not set:
```
make test-e2e-for-splunk

flag needs an argument: -splunk-token

```